utils/__init__.py

```diff
@@ -68,7 +69,7 @@
 BLIND_TEST = 'blind'
 STANDARD_TEST = 'test'
 
-FOLD_NUM_DEFAULT = 5
+FOLD_NUM_DEFAULT = 3
```

Comment:
Let's not change this value in this pull request

Reply:
Changed back to default to 5.